src/rocky/index.js

```diff
@@ -0,0 +1,37 @@
+var rocky = require('rocky');
+
+var dataLogs = [];
+
+// Triggered when memory pressure is an issue
+rocky.on('memorypressure', function(event) {
+  switch(event.level) {
+    case 'high':
+      // Memory pressure is high, free some memory
+      dataLogs = [];
+      console.log('relieved the pressure');
+      break;
+    case 'normal':
+    case 'low':
+      // Not implemented
+      break;
+  }
+});
+
+rocky.on('minutechange', function(event) {
```

Comment:
if the purpose of this test is to regularly hit pressure, doesn't it make more sense to trigger the overflow more often, e.g. `secondchange` and then something line `seconds % 5 == 0` ?